### PR TITLE
project addition by ujjawaltyagii #814

### PR DESCRIPTION
**Pull request by ujjawaltyagii**
## Related Issue

Closes :  #814 

## Description

Added two projects :
1. It's a automation-bot which make things easy for user for accepting the requests and triggering the reaction upon seeing a specific button.

2. A full stack project leetcode like project with bounty of winning Airpods on completion. 

## Screenshots
image

- I am a GSSoC Contributor !

### CODE REVIEW

**Comment by ujjawaltyagii** [comment]
please add gssoc label !

File: src/DB/projects.json [hunk 1]
```diff
@@ -1811,5 +1811,29 @@
...
+        "tech": ["Javascript"]
+      },
+      {
+        "link": "https://github.com/hkirat/peetcode",
+        "title": "PeetCode",
+        "description": "It's an online algorithmic problem solving platform like leetcode where you can run your code. Get a chance to win Airpods by completing bounty.",
+        "tech": ["REACTJS", "CSS", "EXPRESSJS", "HTML"]
+      }
+    ]
   }
```
Comment (priyankarpal) [inline src/DB/projects.json:1838]: ig it's not your profile so kindly add your profile not others

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json (hunk 1 above)
Comment (ujjawaltyagii) [reply src/DB/projects.json:1838]: @priyankarpal  okay ! but these are public projects open for contributions by a infamous youtuber with public profile, with some projects containing even prizes too.
So, we can not add or list any public projects with original profile which are open for contributions ?

Comment (priyankarpal) [reply src/DB/projects.json:1838]: so the goal of projectshut is adding own projects here not others.

Comment (ujjawaltyagii) [reply src/DB/projects.json:1838]: No issue !  will update the PR with my own projects 👍

**Comment by ujjawaltyagii** [comment]
it got messed up while updating the PR ! so closing this one and opening a fresh one !

**Pull request closed without merging**